src/testing/time_refinement_study_advection.h

```diff
@@ -0,0 +1,47 @@
+#ifndef __TIME_REFINEMENT_STUDY_ADVECTION__
+#define __TIME_REFINEMENT_STUDY_ADVECTION__
+
+#include "tests.h"
+#include <deal.II/base/convergence_table.h>
+
+namespace PHiLiP {
+namespace Tests {
+
+/// Advection time refinement study 
+template <int dim, int nstate>
+class TimeRefinementStudyAdvection: public TestsBase
+{
+public:
+    /// Constructor
+    TimeRefinementStudyAdvection(
+            const Parameters::AllParameters *const parameters_input,
+            const dealii::ParameterHandler &parameter_handler_input);
+
+    /// Destructor
+    ~TimeRefinementStudyAdvection() {};
+
+    /// Parameter handler for storing the .prm file being ran
+    const dealii::ParameterHandler &parameter_handler;
+    
+    /// Run test
+    int run_test () const override;
+protected:
+
+    /// Number of times to solve for convergence summary
+    const int n_time_calculations = 4;
+
+    /// Ratio to refine by
+    const double refine_ratio = 0.5;
```

Comment:
Consider making these input parameters and initializing them in the constructor

Reply:
Changed in most recent commit.

Reply:
I still see these in the `time_refinement_study.h` file

Reply:
I saved that file incorrectly. Actually fixed now!

Reply:
They're still being assigned in the header file

Reply:
Yes, the fix will be in my next commit.